README.md

```diff
@@ -223,3 +224,82 @@ No modules.
 ## License
 
 Apache-2.0 Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraform-aws-kms/blob/master/LICENSE).
+
+<!-- BEGIN_TF_DOCS -->
```

Comment:
please use `pre-commit run -a` to format and update docs

Reply:
Running that command got me all passed :) 
```
➜ pre-commit run -a
Terraform fmt............................................................Passed
Terraform validate.......................................................Passed
Terraform docs...........................................................Passed
Terraform validate with tflint...........................................Passed
check for merge conflicts................................................Passed
fix end of files.........................................................Passed
```